_getting-started/index.md

```diff
@@ -7,55 +7,121 @@ includeTOC: true
 
 redirect_from: "/getting-started.html"
 ---
+## Try Scala without installing anything 
+You can [try Scala in your browser](https://scastie.scala-lang.org/MHc7C9iiTbGfeSAvg8CKAA), with access to all Scala compilers and 
+all published libraries.
+
+##  Installing Scala
+### Using the Scala Installer (recommended way)
+The Scala installer is a tool named `cs` that ensures that a JVM and standard Scala tools are installed on your system.  
+
+* Download the `cs` tool and execute the `setup` command
+
+<div class="main-download">
+    <div id="download-step-one">
+        <p>Follow <a href="https://get-coursier.io/docs/cli-overview.html#install-native-launcher" target="_blank">the instructions to install the <code>cs</code> launcher</a> then run:</p>
+        <p><code>$ ./cs setup</code></p>
+    </div>
+</div>
+
+
+Along with managing JVMs, it also installs useful command line tools: 
+ammonite, coursier, scala, scalac, sbt-launcher, scalafmt.
+
+For more information, read [coursier-cli documentation](https://get-coursier.io/docs/cli-overview)
+
+### ...Or manually
+1. if you don't have Java 1.8 or 11 installed, download 
+Java from [Oracle Java 8](https://www.oracle.com/java/technologies/javase-jdk8-downloads.html), [
+Oracle Java 11](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html), 
+or [AdoptOpenJDK 8/11](https://adoptopenjdk.net/). Refer [JDK Compatibility](/overviews/jdk-compatibility/overview.html) for Scala/Java compatibility detail.
+1. Install [sbt](https://www.scala-sbt.org/download.html)
+
+
+## Create a Hello-world project with sbt
+### Using command-line
+sbt is a build tool for Scala. sbt compiles, runs, 
+and tests your projects among other related tasks.
+
+1. `cd` to an empty folder.
+1. Run the following command `sbt new scala/hello-world.g8`.
+This pulls the 'hello-world' template from GitHub.
+It will also create a `target` folder, which you can ignore.
+1. When prompted, name the application `hello-world`. This will
+create a project called "hello-world".
+1. Let's take a look at what just got generated:
+
+```
+- hello-world
+    - project (sbt uses this to install and manage plugins and dependencies)
+        - build.properties
+    - src
+        - main
+            - scala (All of your scala code goes here)
+                - Main.scala (Entry point of program) <-- this is all we need for now
+    - build.sbt (sbt's build definition file)
+```
+
+More documentation about sbt can be found in the [Scala Book](/overviews/scala-book/scala-build-tool-sbt.html) 
+or in the official sbt [documentation](https://www.scala-sbt.org/1.x/docs/index.html)
+
+### Without using the command line
+You can skip the rest of this page and go directly to [Building a Scala Project with IntelliJ and sbt](/getting-started/intellij-track/building-a-scala-project-with-intellij-and-sbt.html#next-steps)
+
+
+## Open hello-world project
```

Comment:
Here too, I would add an introductory sentence explaining that we are going to open the project from a code editor and that the two most popular Scala code editors are IntelliJ and VS Code.